flixel/FlxSprite.hx

```diff
@@ -1207,6 +1225,7 @@ class FlxSprite extends FlxObject
 	
 	private function set_color(Color:FlxColor):Int
 	{
+		Color = 0x00FFFFFF & Color;
```

Comment:
Ah, indeed it is. Another change from a local fix I was working on. It's still the correct logic, because set_color shouldn't be alpha-aware based on its logic later on, there's some edge cases where not doing this causes text to sometimes become transparent as it inherits this function. 

Should I separate it as its own PR?